src/actions/profile-view.js

```diff
@@ -122,6 +161,16 @@ export function changeImplementationFilter(
       getState()
     );
 
+    const { ga } = window;
+    if (ga) {
+      ga('send', {
+        hitType: 'event',
+        eventCategory: 'profile',
+        eventAction: 'change hide platform details',
```

Comment:
not sure this is the right action string ;) "change implementation filter" would be more appropriate, right ?

Reply:
Nice catch, thanks!